iina/PlayerCore.swift

```diff
@@ -2144,6 +2144,7 @@ class PlayerCore: NSObject {
     guard info.state.active else { return }
     info.playSpeed = speed
     sendOSD(.speed(speed))
+    mainWindow.updateSpeedLabel(speed: speed)
```

Comment:
At some point we need to consider breaking all the direct connections between classes. With the addition of the plugin system we probably need to be making more use of events, as seen below:
```swift
postNotification(.iinaSecondSubVisibilityChanged)
```
But no need to do this for this PR. This is an issue for the future. First up should be `MPVController`. That low level class should not be directly calling all of the high level classes that it does today. Because it does so I can't reuse it to have a second mpv core that displays HDR OSC thumbnails.